lib/generators/qa_server/templates/config/authorities/linked_data/scenarios/rda_registry_ld4l_cache_validation.yml

```diff
@@ -319,7 +319,7 @@ search:
     query: polychrome
     subauth: colour_content
     position: 3
-    subject_uri: "http://rdaregistry.info/termList/RDAColourContent/1003"
+    subject_uri: "http://rdaregistry.info/termList/RDACartoDT/1003"
```

Comment:
This is being reverted in PR #391.  With the new indexing approach, this test passes with `RDAColourContent/1003`.  It is marked pending in PR #391 until the new indexing scheme is pushed to the production cache.